laplace/curvature/curvature.py

```diff
@@ -89,6 +89,33 @@ def last_layer_jacobians(model, x):
 
         return Js, f.detach()
 
+    def H_log_likelihood(self, f, sigma_factor=None):
```

Comment:
Might be more appropriate in `GGNInterface` because only `GGN` requires this. But here would also be okay.

Reply:
if we move it to `GGNInterface`, then `BackPackInterface` needs to inherit from it. so would keep it in `CurvatureInterface` for now, let me know if you prefer otherwise

Reply:
That makes sense. It also doesn't hurt to have it in `CurvatureInterface` so lets leave it there.

Reply:
Ok actually I believe this should really go into`GGNInterface` and `BackPackGGN` should then be used for the FunctionalLaplace class (by default). I think it's best because this really corresponds to `GGN`, that's makes the GP come up. If we use EF, we had a different GP, and plain `BackPackInterface` is more of a interface and shouldn't be instantiated if not necessary.